floodviz/templates/index.html

```diff
@@ -18,26 +16,27 @@
 		<div id="hydrograph">
 			<svg></svg>
 		</div>
+		<div id="peakflow_bar"></div>
 		<div id="map"></div>
 	</div>
 
 	<script type="text/javascript">
 		// Global Namespace for project - FV
 		var FV = FV || {};
-		FV.hydrometa = {{ config.HYDRO_META|safe }};
-		FV.mapinfo = {{ mapinfo | tojson |safe }};
```

Comment:
This line was causing an error after pulling from master. I will put it back in.